catalog/ui/catalog-ui-search/src/main/webapp/component/result-item/result-item.view.js

```diff
@@ -163,6 +163,10 @@ define([
                 result.warning = validationWarnings;
             }
 
+            //relevance score
+            result.showRelevanceScore = properties.showRelevanceScores;
```

Comment:
I think it'd be better to do the precision calculation here.  Then you can remove the need for the new helper. 

If we start displaying relevance anywhere else, it might be helpful to calculate the human readable version and store it as the results are parsed (in `Metacard.js`).

Reply:
I removed the helper and moved the calculation here